rasa/shared/nlu/training_data/message.py

```diff
@@ -93,7 +96,9 @@ def build(
         cls,
         text: Text,
         intent: Optional[Text] = None,
-        entities: List[Dict[Text, Any]] = None,
+        entities: Optional[List[Dict[Text, Any]]] = None,
+        intent_metadata: Optional[Any] = None,
+        metadata: Optional[Any] = None,
```

Comment:
also, maybe we can rename `metadata` parameter to `example_metadata` or something similar.